ui/lib/core/addon/components/filter-input.hbs

```diff
@@ -3,16 +3,7 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="field">
-  <p class="control has-icons-left">
-    <input
-      class="filter input"
-      placeholder={{this.placeholder}}
-      data-test-filter-input
-      value={{this.value}}
-      {{on "input" this.onInput}}
-      {{did-insert this.focus}}
-    />
-    <Icon @name="search" class="search-icon has-text-grey-light" />
-  </p>
+<div class="control has-icons-left">
+  <input class="filter input" ...attributes {{on "input" this.onInput}} {{did-insert this.focus}} data-test-filter-input />
+  <Icon @name="search" class="search-icon has-text-grey-light" />
 </div>
```

Comment:
I needed to be able to disable the input and also add a click event listener so I thought it would be better to refactor this to spread attributes on the input element rather than adding args.

Reply:
My only concern is that removing `value` here will not show the initial value passed in. Might look into using the [built-in component](https://guides.emberjs.com/release/components/built-in-components/#toc_input) so the value is automatically passed through `...attributes`

Reply:
I thought about that and can't recall why I didn't go with the built-in component but I have an [assertion that validates that the initial value is set](https://github.com/hashicorp/vault/pull/24247/files#diff-7a7e2fa3b48cb41918ae550e8a9bf8a53cb40d91d7c24a7e499844a6e74fbd53R24) on the input.